pkg/swagger/swagger.go

```diff
@@ -24,7 +29,7 @@ func Run(api, outputDir string) error {
 		Schemes:     []string{"https"},
 		Consumes:    []string{"application/json"},
 		Produces:    []string{"application/json"},
-		Paths:       populateTopLevelPaths("Microsoft.RedHatOpenShift", "openShiftCluster", "OpenShift cluster"),
+		Paths:       g.populateTopLevelPaths("Microsoft.RedHatOpenShift", "openShiftCluster", "OpenShift cluster"),
```

Comment:
Can we put `resourceProviderNamespace` as a parameter into `generator` struct so we don't have to pass it here?

Reply:
Not on this PR, or one if follow-up. Too much change.